### DISCUSSION TITLE
We should host the websites on Cloudflare pages

### DISCUSSION THREAD

**Discussion by Relequestual** [json-schema-org/community#27] [General]
Github pages is fine, Netlify is limiting because you can't have collaborators for free.

Using Cloudflare Pages for hosting would bring us deploy previews and CDN hosting, and allow for unlimited collaborators on the free teir. This would allow others to admin the deploy preview and production deploy config that can't be done via config files or via Netlify admin interface (without paying).

Unless there are any objections, I'll create an issue here for this work in the next week or so.

**Comment by jdesrosiers** [comment]
Sounds good to me.

Anyone out there have experience with Cloudflare pages? Any gotchas we need to be aware of?

**Reply by Relequestual** [reply]
I should clarify, I have tried it and found it as simple as netlify. Hosting https://benhutton.me there.
